eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/catalog/protos/QueryOperationsRequestOrBuilder.java

```diff
@@ -15,15 +15,13 @@
  * limitations under the License.
  */
 
-// Generated by the protocol buffer compiler.  DO NOT EDIT!
-// source: catalog.proto
-
 package org.apache.eventmesh.common.protocol.catalog.protos;
 
 @SuppressWarnings({"all"})
-public interface QueryOperationsRequestOrBuilder extends
-    // @@protoc_insertion_point(interface_extends:eventmesh.catalog.api.protocol.QueryOperationsRequest)
-    com.google.protobuf.MessageOrBuilder {
+public interface QueryOperationsRequestOrBuilder
+        extends
+            // @@protoc_insertion_point(interface_extends:eventmesh.catalog.api.protocol.QueryOperationsRequest)
+            com.google.protobuf.MessageOrBuilder {
 
```

Comment:
May you please tell me why the extends keyword needs to be line-breaking? And why the comment is removed?

Reply:
![image](https://github.com/apache/eventmesh/assets/41445332/95908ffa-8fab-496e-a6ba-b010adaca68a)

Reply:
Thanks for pointing out, automatically generated files under `protos` have been all excluded.